transformer.py

```diff
@@ -206,12 +220,14 @@ def check_continue(transformer: transformer_class.Transformer, check_md: dict, t
             break
 
     # Return the appropriate result
-    return (0) if found_file else (-1, "Unable to find an image file to work with")
+    return 0 if found_file else (-1, "Unable to find an image file to work with")
+
 
-def perform_process(transformer: transformer_class.Transformer, check_md: dict, transformer_md: dict, full_md: dict) -> dict:
+def perform_process(transformer: transformer_class.Transformer, check_md: dict) -> dict:
     """Performs the processing of the data
     Arguments:
         transformer: instance of transformer class
+        check_md:
```

Comment:
Thanks for adding this parameter doc. Please also add for the rest of the parameters (once restored)